types.go

```diff
@@ -89,14 +90,37 @@ func (s *SortedPublicSectorInfo) UnmarshalJSON(b []byte) error {
 
 // NewSortedPrivateSectorInfo returns a SortedPrivateSectorInfo
 func NewSortedPrivateSectorInfo(sectorInfo ...PrivateSectorInfo) SortedPrivateSectorInfo {
-	fn := func(i, j int) bool {
-		return bytes.Compare(sectorInfo[i].SealedCID.Bytes(), sectorInfo[j].SealedCID.Bytes()) == -1
+	var remove_duplicate_privSector = make([]PrivateSectorInfo, 0)
```

Comment:
Maybe use a shorter variable name? We're using it in almost every line of this function. (we can also preallocate this slice)

```suggestion
	result := make([]PrivateSectorInfo, 0, len(sectorInfo))
```